PowerEditor/src/MISC/PluginsManager/PluginsManager.cpp

```diff
@@ -78,11 +78,11 @@ int PluginsManager::loadPlugin(const TCHAR *pluginFilePath, vector<generic_strin
 	    pi->_hLib = ::LoadLibrary(pluginFilePath);
         if (!pi->_hLib)
         {
-            const std::wstring& lastErrorMsg = GetLastErrorAsString();
+            generic_string lastErrorMsg = GetLastErrorAsString();
             if (lastErrorMsg.empty())
                 throw generic_string(TEXT("Load Library is failed.\nMake \"Runtime Library\" setting of this project as \"Multi-threaded(/MT)\" may cure this problem."));

```

Comment:
Another suggestion:
`Load Library has failed.\nChanging the project's \"Runtime Library\" setting to \"Multi-threaded(/MT)\" might solve this problem.`